angular/src/app/app.module.ts

```diff
@@ -0,0 +1,98 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
+import { HttpModule } from '@angular/http';
+import {
+  NgModule
+} from '@angular/core';
+import {
+  RouterModule,
+  PreloadAllModules
+} from '@angular/router';
+
+import { StoreModule } from '@ngrx/store';
+import { EffectsModule } from '@ngrx/effects';
+import { DBModule } from '@ngrx/db';
```

Comment:
Waar wil je IndexedDB voor gaan gebruiken? Of is dit ook een artifact van de starter repo?